pkg/pillar/hypervisor/containerd.go

```diff
@@ -46,7 +46,9 @@ func (ctx ctrdContext) Task(status *types.DomainStatus) types.Task {
 	return ctx
 }
 
-func (ctx ctrdContext) Setup(domainName string, config types.DomainConfig, diskStatusList []types.DiskStatus, aa *types.AssignableAdapters, file *os.File) error {
+func (ctx ctrdContext) Setup(status *types.DomainStatus, config types.DomainConfig, aa *types.AssignableAdapters, file *os.File) error {
```

Comment:
If this doesn't/shouldn't modify status it would be better to pass the struct and not the pointer.

Reply:
Addressed this as part of https://github.com/lf-edge/eve/pull/1271